assets/stylesheets/all.css.scss

```diff
@@ -22,3 +22,10 @@
 
 // Modules
 @import '_modules/**/*';
+
+// Custom
+a.gitter-open-chat-button,  a.gitter-open-chat-button:hover {
```

Comment:
Seems unnecessary to nest for something so small IMO since this is just an override and not part of the greater system.

Only comment would be to remove the `a` selectors since that increase specificity.

Reply:
Yes you're absolutely right. I'd even say it's way more readable when sticking to the plain CSS.